.git-blame-ignore-revs

```diff
@@ -0,0 +1,17 @@
+# [clang-format] Format whole code base
```

Comment:
Maybe a reference and some instructions are nice.

```suggestion
# https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/
#
# To use interactively:
# $ git blame --ignore-revs-file=.git-blame-ignore-revs ...

# [clang-format] Format whole code base
```

Reply:
Thanks, that is a good idea. I added your instructions.